### PR TITLE
Fix EZP-24535: Added comments for ezimage_field parameters

### PR DESCRIPTION
**Pull request by yannickroger**
## Description
In #1300 updating comments was forgotten.

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig [hunk 1]
```diff
@@ -403,6 +403,8 @@
 
 {# This field accepts the following parameters:
  #   - alias (image variation name). Defaults to "original" (e.g. image originally uploaded)
+ #   - parameters.width. Allows forcing width of the image
+ #   - parameters.height. Allow forcing height of the image
```
Comment (crevillo) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig]: Allows here too ;)

**Comment by crevillo** [comment]
+1 with that remark.

**Comment by yannickroger** [comment]
@crevillo done.

**Comment by andrerom** [comment]
+1

File: eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig [hunk 2]
```diff
@@ -403,6 +403,8 @@
 
 {# This field accepts the following parameters:
  #   - alias (image variation name). Defaults to "original" (e.g. image originally uploaded)
+ #   - parameters.width. Allows forcing width of the image
```
Comment (bdunogier) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig]: I'd have specified "in the HTML". It doesn't force the width of the image itself, but of the HTML element that shows the image.

**Comment by bdunogier** [comment]
+1 once the remark is fixed

**Comment by yannickroger** [comment]
@bdunogier Thanks, fixed.
Waiting for travis to merge.

**Comment by bdunogier** [comment]
For a code comment @yannickroger ? I think you can merge without waiting :-)

**Comment by yannickroger** [comment]
Call me superstitious or procedure freak :)

**Comment by andrerom** [comment]
+1 and don't wait for travis for this, actually I stopped it. 1. travis has issues currently and 2. this is purely doc, so not a valid excuse to procrastinate @yannickroger! ;)

Ref:
Compiling procrastination

**Pull request merged**